### ISSUE TITLE
move generated contents to a path you can set

### ISSUE THREAD

**Issue by kjaymiller** [render-engine/render-engine#624]
Cookiecutter by default puts all the contents in a site project folder. This is great for things like documentation but doesn't make sense when the webiste is the main focus of the project. 

### Suggestion

Set the project_path to be "__home". If the variable is not changed, then copytree the folder into the project root.

**Comment by john0isaac** [comment]
We have output_dir from cookiecutter but it always defaults to creating a folder inside the current folder I think we need to work on changing how cookiecutter works as we already have the flag that people can set to specify a folder.
![image](https://github.com/render-engine/render-engine/assets/64026625/4e434e97-b7d3-4b46-b2ad-89d1808d06e9)

**Comment by kjaymiller** [comment]
I think doing a copytree, rmtree would be easier than overriding how cookiecutter works no?

**Comment by john0isaac** [comment]
Yes, it would let's see who takes this on and if they can figure out something better
